src/helpers/utils.ts

```diff
@@ -0,0 +1,9 @@
+export type PartialRequired<T, S extends keyof T> = Omit<Required<T>, S> &
```

Comment:
I tried so hard to _not_ have any utils files... but alas, these are handy functions to have.